DesignDocument/ActivityDiagrams/XML/TrainingModule.xml

```diff
@@ -1 +1 @@
\ No newline at end of file
+<mxfile modified="2018-12-15T13:52:05.001Z" host="www.draw.io" agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.110 Safari/537.36" etag="Yuuu_3MRxmy371ngcc4s" version="9.6.4" type="device"><diagram name="Page-1" id="e7e014a7-5840-1c2e-5031-d8a46d1fe8dd">7Vpdc5s4FP01nuk+pAMIYecxdtxmZ5tOdt3dJH1TQQYlGLFCju3++pVAmA8R4k2MITP1i+FKwuicw7m6MiMwW20/MxQH19TD4cgyvO0IXI4syzRsS3zJyC6LQMPIAj4jnupUBBbkJ85HquiaeDipdOSUhpzE1aBLowi7vBJDjNFNtduShtVfjZGPtcDCRaEevSUeD1TUdM6LhitM/ED99MRysoYfyH30GV1H6vdGFlimn6x5hfJrqYkmAfLophQC8xGYMUp5drTaznAosc1hy8Z9eqZ1f98MR/yQAYqnJxSu1dQ5QyT6IG6UERcnN/KGrVmEVlh8JXFIuDzFa4bCr5hvKHv8QpFHIj/vWW5aoCfV8puaLt/lECcbsgpRJM6mm4BwvIiRK5s2QlH6LNTEnjDjeFsKqVl9xnSFOduJLnlrjvCuqqxNQecYKtKCEpNjqDoiJSF/f+kCRnGgkGxG9QFf3N7fTODVN/fa+Wu+/nQZ+mdQgwB7QnTqlDIeUJ9GKJwX0WmqJCyvaoizgK/EL16a4hBvCb8rHd/LLh+hOHvAnO/UE4XWnIpQce0vlMZqlI6x0kJC18zFLRMBCiGOmI/V2Dty/XBl2n/ai6/f/tl8HwcI/n1mK9bkPFs5YzhEnDxVH76jEwDs0zAgAGW7OzUiPemIHajz0ErY6XmAmr3oFIShcHUJtfDCWAYTMSG+4IjL4JKE4YyGlKW9gZF+ZGfO6CMutSyXquVZ6A4wjzxdQc0scgzLXgG6sgrnHVpFqwO8bBVWXxJtve+ScEOR5rIMniZGjaEq/npKq3DyFo3mrZNqgrP1BLfPgWXROl2JFnSS4JSlKt2aFXst3PZeXeAEirYOVTQYlqL1lR5DkUdXEqwByRpY/ck6ML5fPMzBv1v0O/nDMozbhTk7m7w3VdcE90qZgwPXFqbTq6qBpuoEx5jJtUNZ1AaJ5GRkaSMKEnknkTRzjhMuq1TE0eCEb55S+c3g2hq4KYISuEDim8TYJUuCJZRZ3ZcepKVf73iajlnBEzh2zwnS6qT+GFyCbKhKmjs6w0qQeq2i7CET++AEXjcMaEx6NgzYSa7spW5xDhQxHA9LxI6eD9ETbhGxIeCrOnmcbuQNTNyO2bu4xxq0utr3exklNPJtDRx5/WxqqFYIa5ACoEHa1S5HI6JOJ9scej58s10c7+FWQ28oiXjBjFUv5euQZ36lRhWo//8LZX6mXSilbz+f19dKuvsck+Fjl0Gd82gZdlfwN9di+oJ9BKcxw+JpSHhW+DRkgLQ1GcFLjSzhLLxmZhWHimhUtzMVQiHxI3HqCsixiE+lTxEXhReqYUU8L6W7KZdUJXAE79unhZyZhmLAbvA+q6tawGzYGYfTSObiX3wJLgbH1wHWxgK6+rFOTrM+gvAjrEJ03gDRuQ5RZ/9v5iu0hu0ClOp30yznDZHrTbVA9YX7yhEujZbEF305oZGO9IlXo3bN2PvfSzDPu1g7vb7UattzeJaAF/9pNhvKsFbtdb1Sg3ZdCNZJM/z+tZHjLpnfAe2tO5SdL9Br1SjIH79T0X7UdfTbK6W2NxwOor2VzZdph/3QDsHktLSDgdJuvpL2Vu8eDO11kz817XnRrr98UFsVGWIhj/tfHIEKWh3+bSVOizcjM7SL10/B/D8=</diagram></mxfile>
```

Comment:
The left arrow is bugged and above the right class there is a textfield that should be removed.
Also delete or shrink the title and date to make the file easier to insert in the design document